src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -317,13 +317,25 @@ public static int DivRem(int a, int b, out int result)
             // Restore to using % and / when the JIT is able to eliminate one of the idivs.
             // In the meantime, a * and - is measurably faster than an extra /.
 
+            if (X86Base.IsSupported)
+            {
+                (int quitient, result) = X86Base.DivRem((uint)a, a >> 31, b);
```

Comment:
There lacks a way to represent the `CDQ`/`CQO` instruction. Is there any way better than `>> 31`?